src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/FacebookSignupModalComponentObject.java

```diff
@@ -55,6 +62,29 @@ public void acceptWikiaAppPolicy() {
     }
   }
 
+  public void acceptWikiaAppPolicyNoEmail() {
+
+    Set<String> handles = driver.getWindowHandles();
+
+    if (handles.size() > 1) {
+      for (String winHandle : handles) {
+        //Switch to new window opened
+        driver.switchTo().window(winHandle);
+      }
+      waitForElementByElement(editInfoProvided);
+      editInfoProvided.click();
+      PageObjectLogging.log("acceptWikiaAppPolicyNoEmail", "editing info provided", true);
+      emailCheckbox.click();
```

Comment:
Thank you for the fixes,
It tried to run the test and It still doesn't pass successfully for me, I put comment where it fails.
Is it always passing for you ? It has to always pass otherwise it's just unstable